packages/cjs-module-analyzer/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## 0.1.1 (2021-05-28)
```

Comment:
This is a first release. How does one tease `lerna` to make this `0.1.0`?

Reply:
You've already consumed `0.1.0` because `0.1.0+1-dev` is semantically equal to `0.1.0`.  Lerna is picking the next available version.

Reply:
Maybe you meant to use `0.1.0-dev+1` instead of `0.1.0+1-dev`?